### ISSUE TITLE
Testing 0.6 Release

### ISSUE THREAD

**Issue by wolfbunke** [openego/eTraGo#146]
- Testing installation
  - config.json installed?
  -  PyPSA Fork installed?
  - Packages (ego.io, Pandas, etc.)
- Run test with ` python3 appl.py`
- ...

**Comment by wolfbunke** [comment]
I was testing the release with:

```
virtualenv -p python3 --clear etrago_test 
cd etrago_test/
source bin/activate
pip install etrago --process-dependency-links

```
In the folder etrago_test/lib/python3.6/site-packages/etrago/tools the` config.json` was missing also the pypsa fork was not installed. A Temporary solution can be found here:  http://openego.readthedocs.io/en/dev/getting_started.html

Infos to setup.py:
https://docs.python.org/2/distutils/setupscript.html

After fixing those issues the run and the software works fine.

**Comment by wolfbunke** [comment]
bug is fixed in 40e2ee4
